__tests__/__integration__/dataset/dsNodeActions.integration.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+// tslint:disable:no-magic-numbers
+import * as zowe from "@zowe/cli";
+import { IProfileLoaded } from "@zowe/imperative";
+import * as chai from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import * as sinon from "sinon";
+import * as testConst from "../../../resources/testProfileData";
+import * as vscode from "vscode";
+import * as extension from "../../../src/extension";
+import { ZoweDatasetNode } from "../../../src/ZoweDatasetNode";
+import { DatasetTree } from "../../../src/DatasetTree";
+import * as dsActions from "../../../src/dataset/dsNodeActions";
+
+const TIMEOUT = 45000;
```

Comment:
I think this should be exposed in some sort of test-Configuration constants file.
However I don't think adding more stuff to testProfileData is the correct way to go since it already contains things not related to someone's profile (e.g. patterns).

No change requested, just something to think about while refactoring things. :smile: